src/aap_eda/services/ruleset/activate_rulesets.py

```diff
@@ -297,9 +297,9 @@ def _restart_activation(
         if error:
             seconds = int(settings.ACTIVATION_RESTART_SECONDS_ON_FAILURE)
             msg = (
-                f"Activation {activation.name} failed: {str(error)}, but will "
-                f"retry ({retry_count}/{max_retries}) in {seconds} seconds "
-                "according to its restart policy"
+                f"Activation {activation.name} failed: {str(error)}"
```

Comment:
The problem reported by the jira issue is caused by `{e}` from k8s resolving to an empty string, so it reads "Activation failed: Pod Failed: , but will retry". After the fix, it will read as "Activation failed: Pod Failed: Retry" when `{e}` is empty, else "Activation failed: Pod Failed: bad reasonRetry". They still don't look good to the user.

Reply:
Changed log message to: 
`Activation activation-1 failed: Pod activation-job-3-5bjrp failed with error  , retry (1/5) ...` if e is emtpy
and
`Activation activation-1 failed: Pod activation-job-3-5bjrp failed with error bad reason, retry (1/5) ...` if e is `bad reason`